### PR TITLE
Feat/php metrics

### PR DESCRIPTION
**Pull request by Creadeyh**
Added the PDepend connector to analyze PHP code and save associated metrics.
Added corresponding unit tests and readme details.

### CODE REVIEW

File: connectors/pdepend.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    def analyze_source_code(self):
+        """
+        Analyze the repository by using PHP Depend analysis tool
+        """
+        logging.info('PDepend::analyze_repo')
+        # Test if metrics have been already generated for this version
+        metric = self.session.query(Metric).filter(Metric.version_id == self.version.version_id).first()
+        if not metric:
+            metric = Metric()
+        if self.configuration.language != "PHP":
```
Comment (aurelien-alet) [inline connectors/pdepend.py:39]: don't need this

File: connectors/pdepend.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        nodes = root.findall(f".//{node_name}[@{attribute_name}]")
+        return [node.attrib[attribute_name] for node in nodes]
+    
+    def _list_str_to_int(self, list_):
+        return list(map(int, list_))
+    
+    def __compute_mean(self, xml_root, metric, mean_divide):
+        return Math.get_rounded_mean(
+            self._list_str_to_int(
+                self._xml_get_nodes_atribute(xml_root, mean_divide, metric)
```
Comment (aurelien-alet) [inline connectors/pdepend.py:60]: What if the list is empty -> there is a function in util/math.py now -> get_mean_safe

File: connectors/pdepend.py (hunk 1 above)
Comment (Creadeyh) [reply connectors/pdepend.py:39]: As discussed, the connector now raises an exception if the language is not the right one. Added the same exception to JPeek, CK and Radon connectors.

File: connectors/pdepend.py (hunk 2 above)
Comment (Creadeyh) [reply connectors/pdepend.py:61]: Done

**Pull request merged**